packages/core/blueprints/navi-visualization/files/__root__/navi-visualization-manifests/__name__.js

```diff
@@ -0,0 +1,20 @@
+import ManifestBase from 'navi-core/navi-visualization-manifests/base';
+
+export default class extends ManifestBase {
+  name = '<%= dasherizedModuleName %>';
+
+  niceName = '<%= dasherizedModuleName %>';
+
+  icon = '<%= dasherizedModuleName %>';
```

Comment:
Maybe pick a default icon so something always shows up

Reply:
I am fine with that

Reply:
```suggestion
  icon = 'info';
```